### PR TITLE
Typo Fixes

### PR DESCRIPTION
**Pull request by coliff**
## Launch Checklist

Typo Fixes

 - [x] Confirm **your changes do not include backports from Mapbox projects** (unless with compliant license) - if you are not sure about this, please ask!
 - [x] Briefly describe the changes in this PR.
 - [ ] Link to related issues.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
 - [ ] Write tests for all new functionality.
 - [ ] Document any changes to public APIs.
 - [ ] Post benchmark scores.
 - [x] Add an entry to `CHANGELOG.md` under the `## main` section.

### CODE REVIEW

**Comment by coliff** [comment]
Note: there are many typos in the actual code itself which I haven't changed (because it could break something) but ideally someone should at some point. :-)

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@
 ### 🐞 Bug fixes
 
 - ⚠️ Fix ignoring embedded line breaks when `symbol-placement` is `line` or `line-center` ([#4124](https://github.com/maplibre/maplibre-gl-js/pull/4124))
+- Fix typos in code comments ([#4152](https://github.com/maplibre/maplibre-gl-js/pull/4152)), ([#4156](https://github.com/maplibre/maplibre-gl-js/pull/4156))
```
Comment (HarelM) [inline CHANGELOG.md:11]: Not sure this is interesting enough to write in the changelog...

Comment (coliff) [reply CHANGELOG.md]: no problem, pushing a commit with that removed 👍

**Review by HarelM — APPROVED** [review]

**Comment by coliff** [comment]
I use and recommend this: [marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker)

It's also available as a GitHub Action. I think you could consider adding the `.cspell.json` config in the project root and add the extension as a recommended extension in `.github/extensions`.

You can also run `npx cspell \"**/*.*\" --no-progress` from command-line.

This is the config I used with many false-positives added to words list (to ignore them).

```json
{
  "version": "0.2",
  "words": [
    "Aabb",
    "afterjs",
    "Astro",
    "beforejs",
    "buble",
    "choropleth",
    "CIELAB",
    "codecov",
    "Cpath",
    "Csvg",
    "deduped",
    "Delle",
    "dlopen",
    "documentationjs",
    "easings",
    "EPSG",
    "getx",
    "gety",
    "glfx",
    "GLSL",
    "gzipped",
    "highp",
    "iframes",
    "interp",
    "Jawg",
    "kdbush",
    "libglew",
    "libpango",
    "librsvg",
    "maplibregl",
    "mappedin",
    "MBGL",
    "mediump",
    "mkdocs",
    "mousein",
    "mouseup",
    "mozfullscreenchange",
    "MSAA",
    "msvs",
    "mult",
    "nise",
    "nofollow",
    "noopener",
    "Noto",
    "onmozfullscreenchange",
    "onmsfullscreenchange",
    "orta",
    "OSMUS",
    "OTLS",
    "outofmaxbounds",
    "pango",
    "perp",
    "plantuml",
    "plpopup",
    "PLSS",
    "pngjs",
    "pygments",
    "qrcode",
    "Rebranded",
    "rfid",
    "rgba",
    "rtoggle",
    "sdfgamma",
    "slugg",
    "splc",
    "Struct",
    "tcpopup",
    "testem",
    "threejs",
    "tinysdf",
    "tipopup",
    "trfilter",
    "trheader",
    "trpopup",
    "tsdoc",
    "tsnode",
    "tstypes",
    "unassert",
    "unblended",
    "unbundle",
    "unflipped",
    "unitbezier",
    "unminified",
    "Unpitched",
    "unproject",
    "unrotated",
    "unthrottled",
    "untransitioned",
    "unversioned",
    "verticalizable",
    "verticalized",
    "WCAG",
    "webgl",
    "webglcontextcreationerror",
    "webglcontextlost",
    "webglcontextrestored",
    "webp",
    "whoots",
    "wrsheader",
    "wrslegend",
    "wrslist",
    "Xfrom",
    "xlink",
    "xvfb",
    "Zfrom"
  ],
  "allowCompoundWords": true,
  "language": "en,en-GB,en-US",
  "ignorePaths": [
    "**/*.geojson"
  ],
  "ignoreRegExpList": [
    "@\\w+",
    "\\[\\w+"
  ],
  "useGitignore": true
}
```

**Pull request merged**